Source/DafnyCore/Dafny.atg

```diff
@@ -2008,7 +2007,7 @@ MethodDecl<DeclModifierData dmod, bool allowConstructor, out Method/*!*/ m>
   ]
   (. if (!hasName) {
        if (!isConstructor) {
-         SemErr(ErrorID.p_method_missing_name, la, "a method must be given a name (expecting identifier)");
+         SemErr(ParseErrors.ErrorId.p_method_missing_name, la, "a method must be given a name (expecting identifier)");
```

Comment:
Could you import ErrorId so that you don't need to repeat ParseErrors.ErrorId, just ErrorId? You could also consider renaming ParseErrorId if you report things other than parse errors.